finetune/base.py

```diff
@@ -142,9 +138,17 @@ def _initialize(self):
         self.saver = Saver(
             fallback_filename=self.config.base_model_path,
             exclude_matches=None if self.config.save_adam_vars else "Adam",
-            variable_transforms=[embedding_preprocessor(self.input_pipeline, self.config)],
             save_dtype=self.config.save_dtype,
-            target_model_init_from_base_model=self.config.target_model_init_from_base_model
+        )
+
+    def init_from_checkpoint(self, checkpoint_path):
```

Comment:
Could we do:

```
def load_from_checkpoint(cls, checkpoint_path):
    model = cls.load(checkpoint_path)
    model.init_from_checkpoint(checkpoint_path)
    return model
```

or am I totally missing how this fns right now?

Reply:
checkpoint path is not in the same format as the .save/.load format. we could do something like
```
def load_from_checkpoint(cls, checkpoint_path, **config):
     model = cls(**config)
     model.init_from_checkpoint(checkpoint_path)
     return model
```